src/components/projects/Entry.jsx

```diff
@@ -68,112 +76,195 @@ function Entry() {
+              <Typography align="justify" gutterBottom variant="body1">
+                Title
+              </Typography>
+              <TextField
+                className={classes.textInput}
+                data-testid="input-field"
+                error={hasErrorName}
+                helperText={hasErrorName ? formik.errors.name : ""}
+                id="name"
+                label="Name"
+                name="name"
+                onBlur={formik.handleBlur}
+                onChange={formik.handleChange}
+                type="text"
+                value={formik.values.name}
+                variant="outlined"
+              />
+            </div>
+            <div className="u-mb-1">
+              <Typography align="justify" gutterBottom variant="body1">
+                Owner Contact
+              </Typography>
+              <TextField
+                InputProps={{
+                  readOnly: true,
+                }}
+                className={classes.textInput}
+                data-testid="input-field"
+                helperText="This is the owner contact"
+                id="contact"
+                name="contact"
+                type="email"
+                value={formik.values.contact}
+                variant="outlined"
+              />
+            </div>
+            <div className="u-mb-1">
+              <Typography align="justify" gutterBottom variant="body1">
+                Add Description
+              </Typography>
+              <TextField
+                className={classes.textInput}
+                data-testid="input-field"
+                error={hasErrorDescription}
+                helperText={
+                  hasErrorDescription ? formik.errors.description : ""
+                }
+                id="description"
+                label="Description"
+                multiline
+                name="description"
+                onBlur={formik.handleBlur}
+                onChange={formik.handleChange}
+                rows={4}
+                value={formik.values.description}
+                variant="outlined"
+              />
+            </div>
+            <div className="u-mb-1">
+              <Typography align="justify" gutterBottom variant="body1">
+                Invitation Text
+              </Typography>
+              <TextField
+                className={classes.textInput}
+                data-testid="input-field"
+                error={hasErrorTextInvitation}
+                helperText={
+                  hasErrorTextInvitation ? formik.errors.textInvitation : ""
+                }
+                id="textInvitation"
+                label="Text invitation"
+                multiline
+                name="textInvitation"
+                onBlur={formik.handleBlur}
+                onChange={formik.handleChange}
+                rows={2}
+                value={formik.values.textInvitation}
+                variant="outlined"
+              />
+            </div>
```

Comment:
Remove this field, it's work in progress too

Reply:
In backend it is already implemented, Alejandro and Jorge agreed with it. Is that ok for you?